src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -7,15 +7,10 @@
 
 helps['consumption'] = """
     type: group
-    short-summary: Manage consumption of Azure resources.
-"""
-
-helps['consumption usage'] = """
-    type: group
-    short-summary: Inspect the usage of Azure resources.
```

Comment:
Removing this just means there's now no help message for the `az consumption usage` group.
Either add this back in or change the command to `az consumption list-usage` as there's only a single command in that group.

Reply:
I understand what you say. But when I look at the current documentation. It says "List of commands" and shows "az consumption usage" and "az consumption usage list". But there is only one command in reality and that is az consumption usage list
And to your comment to get rid of the group if we are not planning more commands under usage, yes we do plan to have more commands under usage coming soon.
So the question is, how to keep the group but not list as a command in the help documentation?

Reply:
@derekbekoe Is this a CLI bug regarding groups with only one command, or a feature?